app/models/agents/website_agent.rb

```diff
@@ -259,13 +261,24 @@ def check_urls(in_url)
       end
     end
 
+    def get_valid_url(url)
+      if (url =~ URI::ABS_URI) 
+      	return url
```

Comment:
Yes, I'm going under the assumption that URI is valid if it passes that check. 

Relative URLs work. 

And that line should be changed to 
`if (url =~ URI::ABS_URI or url =~ URI::REL_URI)`
to bypass Addressable.